people-and-planet-ai/geospatial-classification/requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+earthengine-api==0.1.292
```

Comment:
The requirements file is only for the serving app which doesn't need all of this. Should I have a separate requirements file for the whole project?

Reply:
Mmm that's a good question. I think the requirements.txt file should contain anything required to run the sample locally. And then we could have another requirements for the serving app. It doesn't matter if there's duplication since we have a bot that auto-updates requirements.

Reply:
Ok, sounds good! Will do that